app/views/forbiddenError.scala.html

```diff
@@ -0,0 +1,16 @@
+@*
+* This is a placeholder to allow us to write unit tests which return an error page.
+* It will be replaced once we have a properly designed error page
+*@
+@(errorMessage: String)(implicit messages: Messages)
+
+@main(Messages("error.header")) {
+    @defining(play.core.PlayVersion.current) { version =>
+        <div class="govuk-grid-row">
+            <div class="govuk-grid-column-two-thirds">
+                <h1 class="govuk-heading-l">You are not permitted to see this page</h1>
```

Comment:
Do you want a hand with moving this text into the messages file? @techncl might be a good person to pair with, since he's done this quite a lot recently.